### PR TITLE
replace all get_dist with GET_DIST, this is a good idea!

### LINKED ISSUE #9861: pods say all beacons are 125 meters away
### Labels

[BUG]

### Description

same as title

### Steps to reproduce

1.play pod wars
2.click activate sensors
3.scan area
4. all of them are 125M away

### Expected Behavior

proper distance

### Additional Information & Screenshots

Reported by: ZWRh3
On server: Goonstation 1 Classic: Heisenbee
Round log date: 2022-07-25-12-45
Reported on: 2022-07-25 13:08:03

### PR DESCRIPTION
**Pull request by zjdtmkhzt**
## About the PR 
Replaces all the get_dist with GET_DIST, which can actually get results above 127 and work in containers or something.

Still need to go over things because there are a few victims of search-replace
## Why's this needed? 
idk, I was told we don't use get_dist, but we actually kind of do, quite a lot? /shrug

fixes #9861

### CODE REVIEW

File: code/WorkInProgress/SpyGuyStuff.dm [hunk 1]
```diff
@@ -677,7 +677,7 @@ proc/Create_Tommyname()
 	for(var/i = created_atoms.len; i > 0; i--)
 		var/atom/A = created_atoms[i]
 		if(istype(A, /obj/machinery/power/solar) || istype(A, /turf/simulated/floor/airless/solar))
-			var/dist = get_dist_from_centreline(A)
+			var/dist = GET_DIST_from_centreline(A)
```
Comment (zjdtmkhzt) [inline code/WorkInProgress/SpyGuyStuff.dm:680]:
```suggestion
			var/dist = get_dist_from_centreline(A)
```

File: code/WorkInProgress/SpyGuyStuff.dm [hunk 2]
```diff
@@ -785,13 +785,13 @@ proc/Create_Tommyname()
 	//Look, I'm Swedish, I don't know your goddamn mathwords
 	return d
 
-/obj/solar_control/proc/get_dist_from_centreline(var/atom/A) //Finds the distance from the closest point on the extension line
+/obj/solar_control/proc/GET_DIST_from_centreline(var/atom/A) //Finds the distance from the closest point on the extension line
```
Comment (zjdtmkhzt) [inline code/WorkInProgress/SpyGuyStuff.dm:788]:
```suggestion
/obj/solar_control/proc/get_dist_from_centreline(var/atom/A) //Finds the distance from the closest point on the extension line
```

File: code/WorkInProgress/SpyGuyStuff.dm [hunk 3]
```diff
@@ -785,13 +785,13 @@ proc/Create_Tommyname()
...
 	if(extension_dir & (NORTH|SOUTH) )
 		.= abs(A.x - src.x)
 	else if ( extension_dir & (EAST|WEST) )
 		.= abs(A.y - src.y)
 
-	DEBUG_MESSAGE("get_dist from [log_loc(A)] returned: [.]")
+	DEBUG_MESSAGE("GET_DIST from [log_loc(A)] returned: [.]")
```
Comment (zjdtmkhzt) [inline code/WorkInProgress/SpyGuyStuff.dm:794]:
```suggestion
	DEBUG_MESSAGE("get_dist from [log_loc(A)] returned: [.]")
```

File: code/modules/items/specials/ItemSpecials.dm [hunk 4]
```diff
@@ -86,7 +86,7 @@
 
 #define ITEMSPECIAL_PIXELDIST_SQUARED  (70 * 70) //lol i'm putting the define RIGHT HERE.
 // These two numbers will be compared later (pixeldist squared AND the result of this function). We don't need to do unnessecary sqrt cause this is just a simple < > comparison!
-/proc/get_dist_pixel_squared(var/atom/source, var/atom/target, params)
+/proc/GET_DIST_pixel_squared(var/atom/source, var/atom/target, params)
```
Comment (zjdtmkhzt) [inline code/modules/items/specials/ItemSpecials.dm:89]:
```suggestion
/proc/get_dist_pixel_squared(var/atom/source, var/atom/target, params)
```

File: code/modules/items/specials/ItemSpecials.dm [hunk 5]
```diff
@@ -295,7 +295,7 @@
 		if(!isturf(target.loc) && !isturf(target)) return
 		if(!usable()) return
 		var/list/parameters = params2list(params)
-		if(parameters["left"] && master && get_dist_pixel_squared(usr, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
+		if(parameters["left"] && master && GET_DIST_pixel_squared(usr, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
```
Comment (zjdtmkhzt) [inline code/modules/items/specials/ItemSpecials.dm:298]:
```suggestion
		if(parameters["left"] && master && get_dist_pixel_squared(usr, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
```

File: code/modules/items/specials/ItemSpecials.dm [hunk 6]
```diff
@@ -408,7 +408,7 @@
 		if(!usable()) return
 		if(!isturf(target.loc) && !isturf(target)) return
 		var/list/parameters = params2list(params)
-		if(parameters["left"] && master && get_dist_pixel_squared(usr, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
+		if(parameters["left"] && master && GET_DIST_pixel_squared(usr, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
```
Comment (zjdtmkhzt) [inline code/modules/items/specials/ItemSpecials.dm:411]:
```suggestion
		if(parameters["left"] && master && get_dist_pixel_squared(usr, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
```

File: code/modules/items/specials/ItemSpecials.dm [hunk 7]
```diff
@@ -438,7 +438,7 @@
 	pixelaction(atom/target, params, mob/user, reach)
 		if(!isturf(target.loc) && !isturf(target)) return
 		if(!usable(user)) return
-		if(params["left"] && master && get_dist_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
+		if(params["left"] && master && GET_DIST_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
```
Comment (zjdtmkhzt) [inline code/modules/items/specials/ItemSpecials.dm:441]:
```suggestion
		if(params["left"] && master && get_dist_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
```

File: code/modules/items/specials/ItemSpecials.dm [hunk 8]
```diff
@@ -496,7 +496,7 @@
...
```
Comment (zjdtmkhzt) [inline code/modules/items/specials/ItemSpecials.dm:499]:
```suggestion
		if(params["left"] && master && get_dist_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
```

File: code/modules/items/specials/ItemSpecials.dm [hunk 9]
```diff
@@ -593,7 +593,7 @@
 	pixelaction(atom/target, list/params, mob/user, reach)
 		if(!isturf(target.loc) && !isturf(target)) return
 		if(!usable(user)) return
-		if(params["left"] && master && get_dist_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
+		if(params["left"] && master && GET_DIST_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
```
Comment (zjdtmkhzt) [inline code/modules/items/specials/ItemSpecials.dm:596]:
```suggestion
		if(params["left"] && master && get_dist_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
```

File: code/modules/items/specials/ItemSpecials.dm [hunk 10]
```diff
@@ -696,7 +696,7 @@
 		var/turf/T = get_turf(target)
 		if(!T) return
 		if(!usable(user)) return
-		if(params["left"] && master && get_dist_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
+		if(params["left"] && master && GET_DIST_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
```
Comment (zjdtmkhzt) [inline code/modules/items/specials/ItemSpecials.dm:699]:
```suggestion
		if(params["left"] && master && get_dist_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
```

File: code/modules/items/specials/ItemSpecials.dm [hunk 11]
```diff
@@ -756,7 +756,7 @@
 		if(!isturf(target.loc) && !isturf(target)) return
 		if(!usable(user)) return
 
-		if(params["left"] && (master || user) && get_dist_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
+		if(params["left"] && (master || user) && GET_DIST_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
```
Comment (zjdtmkhzt) [inline code/modules/items/specials/ItemSpecials.dm:759]:
```suggestion
		if(params["left"] && (master || user) && get_dist_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
```

File: code/modules/items/specials/ItemSpecials.dm [hunk 12]
```diff
@@ -804,7 +804,7 @@
 		if(!isturf(target.loc) && !isturf(target)) return
 		if(!usable(user)) return
 
-		if(params["left"] && get_dist_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
+		if(params["left"] && GET_DIST_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
```
Comment (zjdtmkhzt) [inline code/modules/items/specials/ItemSpecials.dm:807]:
```suggestion
		if(params["left"] && get_dist_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
```

File: code/modules/items/specials/ItemSpecials.dm [hunk 13]
```diff
@@ -859,7 +859,7 @@
 		if(!isturf(target.loc) && !isturf(target)) return
 		if(!usable()) return
 		var/list/parameters = params2list(params)
-		if(parameters["left"] && master && get_dist_pixel_squared(usr, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
+		if(parameters["left"] && master && GET_DIST_pixel_squared(usr, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
```
Comment (zjdtmkhzt) [inline code/modules/items/specials/ItemSpecials.dm:862]:
```suggestion
		if(parameters["left"] && master && get_dist_pixel_squared(usr, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
```

File: code/modules/items/specials/ItemSpecials.dm [hunk 14]
```diff
@@ -906,7 +906,7 @@
 	pixelaction(atom/target, params, mob/user, reach)
 		if(!isturf(target.loc) && !isturf(target)) return
 		if(!usable(user)) return
-		if(params["left"] && get_dist_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
+		if(params["left"] && GET_DIST_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
```
Comment (zjdtmkhzt) [inline code/modules/items/specials/ItemSpecials.dm:909]:
```suggestion
		if(params["left"] && get_dist_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
```

File: code/modules/items/specials/ItemSpecials.dm [hunk 15]
```diff
@@ -954,7 +954,7 @@
...
```
Comment (zjdtmkhzt) [inline code/modules/items/specials/ItemSpecials.dm:957]:
```suggestion
		if(params["left"] && get_dist_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
```

File: code/modules/items/specials/ItemSpecials.dm [hunk 16]
```diff
@@ -993,7 +993,7 @@
...
```
Comment (zjdtmkhzt) [inline code/modules/items/specials/ItemSpecials.dm:996]:
```suggestion
		if(params["left"] && get_dist_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
```

File: code/modules/items/specials/ItemSpecials.dm [hunk 17]
```diff
@@ -1043,7 +1043,7 @@ ABSTRACT_TYPE(/datum/item_special/spark)
 		if(!isturf(target.loc) && !isturf(target)) return
 		if(!usable(user)) return
 
-		if(params["left"] && master && get_dist_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
+		if(params["left"] && master && GET_DIST_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
```
Comment (zjdtmkhzt) [inline code/modules/items/specials/ItemSpecials.dm:1046]:
```suggestion
		if(params["left"] && master && get_dist_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
```

File: code/modules/items/specials/ItemSpecials.dm [hunk 18]
```diff
@@ -1094,7 +1094,7 @@ ABSTRACT_TYPE(/datum/item_special/spark)
 /datum/item_special/spark/baton
 	pixelaction(atom/target, params, mob/user, reach)
 		if(user.a_intent != INTENT_DISARM) return //only want this to deploy on disarm intent
-		if(!istype(master, /obj/item/baton) || get_dist_pixel_squared(user, target, params) <= ITEMSPECIAL_PIXELDIST_SQUARED) return
+		if(!istype(master, /obj/item/baton) || GET_DIST_pixel_squared(user, target, params) <= ITEMSPECIAL_PIXELDIST_SQUARED) return
```
Comment (zjdtmkhzt) [inline code/modules/items/specials/ItemSpecials.dm:1097]:
```suggestion
		if(!istype(master, /obj/item/baton) || get_dist_pixel_squared(user, target, params) <= ITEMSPECIAL_PIXELDIST_SQUARED) return
```

File: code/modules/items/specials/ItemSpecials.dm [hunk 19]
```diff
@@ -1106,7 +1106,7 @@ ABSTRACT_TYPE(/datum/item_special/spark)
 /datum/item_special/spark/gloves
 	pixelaction(atom/target, params, mob/user, reach)
 		..()
-		if(!istype(master, /obj/item/clothing/gloves) || get_dist_pixel_squared(user, target, params) <= ITEMSPECIAL_PIXELDIST_SQUARED) return
+		if(!istype(master, /obj/item/clothing/gloves) || GET_DIST_pixel_squared(user, target, params) <= ITEMSPECIAL_PIXELDIST_SQUARED) return
```
Comment (zjdtmkhzt) [inline code/modules/items/specials/ItemSpecials.dm:1109]:
```suggestion
		if(!istype(master, /obj/item/clothing/gloves) || get_dist_pixel_squared(user, target, params) <= ITEMSPECIAL_PIXELDIST_SQUARED) return
```

File: code/modules/items/specials/ItemSpecials.dm [hunk 20]
```diff
@@ -1143,7 +1143,7 @@ ABSTRACT_TYPE(/datum/item_special/spark)
 	pixelaction(atom/target, params, mob/user, reach)
 		if(!isturf(target.loc) && !isturf(target)) return
 		if(!usable(user)) return
-		if(params["left"] && master && get_dist_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
+		if(params["left"] && master && GET_DIST_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
```
Comment (zjdtmkhzt) [inline code/modules/items/specials/ItemSpecials.dm:1146]:
```suggestion
		if(params["left"] && master && get_dist_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
```

File: code/modules/items/specials/ItemSpecials.dm [hunk 21]
```diff
@@ -1200,7 +1200,7 @@ ABSTRACT_TYPE(/datum/item_special/spark)
...
```
Comment (zjdtmkhzt) [inline code/modules/items/specials/ItemSpecials.dm:1203]:
```suggestion
		if(params["left"] && master && get_dist_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
```

File: code/modules/items/specials/ItemSpecials.dm [hunk 22]
```diff
@@ -1264,7 +1264,7 @@ ABSTRACT_TYPE(/datum/item_special/spark)
...
```
Comment (zjdtmkhzt) [inline code/modules/items/specials/ItemSpecials.dm:1267]:
```suggestion
		if(params["left"] && master && get_dist_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
```

File: code/modules/items/specials/ItemSpecials.dm [hunk 23]
```diff
@@ -1346,7 +1346,7 @@ ABSTRACT_TYPE(/datum/item_special/spark)
...
```
Comment (zjdtmkhzt) [inline code/modules/items/specials/ItemSpecials.dm:1349]:
```suggestion
		if(params["left"] && master && get_dist_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
```

File: code/modules/items/specials/ItemSpecials.dm [hunk 24]
```diff
@@ -1391,7 +1391,7 @@ ABSTRACT_TYPE(/datum/item_special/spark)
 		if(!isturf(target.loc) && !isturf(target)) return
 		if(!usable(user)) return
 
-		if(params["left"] && master && get_dist_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
+		if(params["left"] && master && GET_DIST_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
```
Comment (zjdtmkhzt) [inline code/modules/items/specials/ItemSpecials.dm:1394]:
```suggestion
		if(params["left"] && master && get_dist_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
```

File: code/modules/items/specials/ItemSpecials.dm [hunk 25]
```diff
@@ -1480,7 +1480,7 @@ ABSTRACT_TYPE(/datum/item_special/spark)
 		if(!isturf(target.loc) && !isturf(target)) return
 		if(!usable(user)) return
 
-		if(params["left"] && (master && K) && get_dist_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
+		if(params["left"] && (master && K) && GET_DIST_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
```
Comment (zjdtmkhzt) [inline code/modules/items/specials/ItemSpecials.dm:1483]:
```suggestion
		if(params["left"] && (master && K) && get_dist_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
```

File: code/modules/items/specials/ItemSpecials.dm [hunk 26]
```diff
@@ -1608,7 +1608,7 @@ ABSTRACT_TYPE(/datum/item_special/spark)
 		if(!isturf(target.loc) && !isturf(target)) return
 		if(!usable(user)) return
 
-		if(params["left"] && params["ai"] || get_dist_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
+		if(params["left"] && params["ai"] || GET_DIST_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
```
Comment (zjdtmkhzt) [inline code/modules/items/specials/ItemSpecials.dm:1611]:
```suggestion
		if(params["left"] && params["ai"] || get_dist_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
```

File: code/modules/items/specials/ItemSpecials.dm [hunk 27]
```diff
@@ -1711,7 +1711,7 @@ ABSTRACT_TYPE(/datum/item_special/spark)
 	pixelaction(atom/target, params, mob/user, reach)
 		if(!isturf(target.loc) && !isturf(target)) return
 		if(!usable(user)) return
-		if(params["left"] && master && get_dist_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
+		if(params["left"] && master && GET_DIST_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
```
Comment (zjdtmkhzt) [inline code/modules/items/specials/ItemSpecials.dm:1714]:
```suggestion
		if(params["left"] && master && get_dist_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
```

File: code/modules/items/specials/ItemSpecials.dm [hunk 28]
```diff
@@ -1771,7 +1771,7 @@ ABSTRACT_TYPE(/datum/item_special/spark)
...
```
Comment (zjdtmkhzt) [inline code/modules/items/specials/ItemSpecials.dm:1774]:
```suggestion
		if(params["left"] && master && get_dist_pixel_squared(user, target, params) > ITEMSPECIAL_PIXELDIST_SQUARED)
```

**Review by zjdtmkhzt — COMMENTED** [review]
woops

**Comment by ZWRh3** [comment]
iirc this also patches a bug with it allowing to pass the distance check for objects on a different z level

**Comment by zjdtmkhzt** [comment]
Hmm, I hope that isn't intended for any of those then...

**Review by frawhst — APPROVED** [review]

**Comment by frawhst** [comment]
let's see if this breaks anything

**Comment by zjdtmkhzt** [comment]
People are now gibbed when being thrown into walls.

**Pull request merged**